src/sb/app/membership/data.cljc

```diff
@@ -239,3 +238,20 @@
   (into []
         (map #(->> (:tag/id %) (resolve-tag (:membership/entity membership)) (:tag/color)))
         (:entity/tags membership)))
+
+(defn members
```

Comment:
could we add docstrings and maybe change the argument names for `members` and `member-of` to make it more clear what is happening? 

eg. something like:

`members`: "Returns members of `entity`, with an optional transducer (`xform`) applied to the memberships first."

`member-of`: "Returns all entities that `member` is a member of, with an optional transducer (`xform`) applied to the memberships first." (change arg name to `member`)

`into` will be eager, `eduction` would be lazy, if we typically use all the values then I think `into` is preferred (as laziness can cause unexpected issues with reactivity & is a bit less performant)